git/Cheatsheat.md

```diff
@@ -0,0 +1,100 @@
+# Git Cheatsheet
+
+This file contains guides such as **how to write good commit message**, **useful git commands** and their explanation. We'll start with **Writing Good Commit Message** section that contains [Erlang's guide](https://github.com/erlang/otp/wiki/writing-good-commit-messages) in a nutshell. Next section, **Useful Git commands and how to use them**, lists useful Git commands. Section **How to rebase properly?** explains process of rebasing which is required in proposed Git flow given in [Contributing](https://github.com/dmarjanovic94/guides/blob/master/git/Contributing.md) chapter.
+
+## Table of Contents
+
+- [Writing good commit message](https://github.com/lambdaworkshub/contributing-guide/blob/master/git/Cheatsheat.md#writing-good-commit-message)
+- [Useful Git commands and how to use them](https://github.com/lambdaworkshub/contributing-guide/blob/master/git/Cheatsheat.md#useful-git-commands-and-how-to-use-them)
+- [How to rebase properly?](https://github.com/lambdaworkshub/contributing-guide/blob/master/git/Cheatsheat.md#how-to-rebase-properly)
+
+### Writing Good Commit Message
+
+Provided [Erlang's guide](https://github.com/erlang/otp/wiki/writing-good-commit-messages) gives rules for writing good commit messages. There are listed most important rules from above mentioned guide.
```

Comment:
Use this sentence `The most important rules from the above-mentioned guide are listed below.`